scripts/patrol/patrol_outcome.py

```diff
@@ -734,9 +736,11 @@ def _handle_herbs(self, patrol:'Patrol') -> str:
                 insert = f"{specfic_herbs[0]} and {specfic_herbs[1]} were"
         else:
             insert = f"{', '.join(specfic_herbs[:-1])}, and {specfic_herbs[-1]} were"
+
+        insert = re.sub("[_]", " ", insert)
         
         game.herb_events_list.append(f"{insert.capitalize()} gathered on a patrol.")
-        return f"{insert.capitalize()} gathered"
+        return f"{insert.capitalize()} gathered on this patrol."
```

Comment:
This seems unnecessarily wordy.  No other results text includes "on this patrol."

Reply:
hmm yeah you might be right. It looked nice to me while playing but it might be a bit much